### PR TITLE
Changed storageClassName gp2 to storage-class-name

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/issues/22267

### CODE REVIEW

**Comment by mburke5678** [comment]
@anpingli Is this something you can review?

**Review by anpingli — COMMENTED** [review]
LGTM

File: modules/cluster-logging-deploy-cli.adoc [hunk 1]
```diff
@@ -267,7 +267,7 @@ spec:
     elasticsearch:
       nodeCount: 3
       storage:
-        storageClassName: "gp2"
+        storageClassName: "<storage-class-name>"
```
Comment (kalexand-rh) [inline modules/cluster-logging-deploy-cli.adoc:270]: Is this variable clear enough without the annotation? And will someone on dedicated have this available to specify?

File: modules/cluster-logging-deploy-cli.adoc [hunk 2]
```diff
@@ -332,7 +332,7 @@ spec:
 However, an unmanaged deployment does not receive updates until Cluster Logging is placed back into the `Managed` state.
 <3> Settings for configuring Elasticsearch. Using the Custom Resource (CR), you can configure shard replication policy and persistent storage.
 <4> Specify the number of Elasticsearch nodes. See the note that follows this list.
-<5> Specify that each Elasticsearch node in the cluster is bound to a Persistent Volume Claim.
+<5> Enter the name of an existing StorageClass for Elasticsearch storage. The StorageClass should allocate block storage for best performance.
```
Comment (kalexand-rh) [inline modules/cluster-logging-deploy-cli.adoc:335]: > ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.

Maybe:
> For best performance, specify a StorageClass that allocates block storage.

**Review by kalexand-rh — COMMENTED** [review]
One question, one suggestion, and it LGTM.

File: modules/cluster-logging-deploy-cli.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/cluster-logging-deploy-cli.adoc:270]: Yes. Was going to ask about Dedicated after I got the wording down.

File: modules/cluster-logging-deploy-cli.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/cluster-logging-deploy-cli.adoc:335]: I was thinking that too. Thanks!

Comment (lamek) [reply modules/cluster-logging-deploy-cli.adoc:335]: We should update this for OSD as well. OSD customers can now utilize AWS or GCP storage depending on their platform. That said, we do have a larger audit/cleanup story to address the issue of where we are applying and managing filtering like this for different distros.

**Comment by mburke5678** [comment]
@lamek PTAL for Dedicated changes.

**Pull request merged**